api/worker.go

```diff
@@ -21,6 +21,13 @@ type RHPScanRequest struct {
 	Timeout time.Duration   `json:"timeout"`
 }
 
+// RHPPriceTableRequest is the request type for the /rhp/pricetable endpoint.
+type RHPPriceTableRequest struct {
+	HostKey    types.PublicKey `json:"hostKey"`
+	SiamuxAddr string          `json:"siamuxAddr"`
+	Timeout    time.Duration   `json:"timeout"`
```

Comment:
Why provide the timeout here instead of applying it in the http client through a context?